index.src.html

```diff
@@ -394,6 +435,233 @@
+              as:
+            </p>
+            <p>
+              `r2 = px^2 + py^2`
+            </p>
+            <p>
+              `r = 1 + k1 * r2 + k2 * r2^2 + k3 * r2^3`
+            </p>
+            <p>
+              `Xd = dz * (px * r + 2 * p1 * px * py + p2 * (r2 + 2 * px^2))`
+            </p>
+            <p>
+              `Yd = dz * (py * r + 2 * p2 * px * py + p1 * (r2 + 2 * py^2))`
+            </p>
+            <p>
+              `Zd = dz`
+            </p>
+          </li>
+        </ul>
+        <p class="example">
+          In code, this condition would be like
+          <code>if(track.getSettings().deprojectionDistortionCoefficients)</code>
+        </p>
+      </section>
+      <section>
+        <h2>
+          <dfn>Transform</dfn> from depth to color 3D space
+        </h2>
+        <p>
+          The 3D point (Xd, Yd, Zd), that is a result of <a>project</a> depth
+          value to 3D point step, is in depth camera 3D coordinate system. To
+          get the coordinates of the same point in space, but in color camera
+          3D coordinate system, we use matrix multiplication of
+          <a>transformation from depth to video</a> matrix by the (Xd, Yd, Zd)
+          3D point vector.
+        </p>
+        <p>
+          Let (Xc, Yc, Zc) be the output of this step - a 3D coordinates of
+          projected <a>depth map value</a> to color camera 3D space.
+        </p>
+        <p>
+          Let <var>M</var> be <a>transformation matrix</a> defined in <a>depth
+          map</a>'s <a>depthToVideoTransform</a> field.
+        </p>
+        <p>
+          To multiply 4x4 matrix by 3 element vector, we extend the 3D vector
+          by one element to 4 dimensional vector. After multiplication, we use
+          vector's x, y and z coordinates as the result.
+        </p>
+        <p>
+          `((Xc), (Yc), (Zc)) = ([M] xx ((Xd), (Yd), (Zd), (1))).xyz`
+        </p>
+      </section>
+      <section>
+        <h2>
+          <dfn>Project</dfn> from color 3D to pixel
+        </h2>
+        <p>
+          With this step, we conclude mapping of a <a>depth map value</a>
+          coordinate to a pixel value in color video frame.
+        </p>
+        <p>
+          For projection we use the corresponding color track's <a href=
+          "#mediatracksettings-dictionary">MediaTrackSettings</a>. We get the
+          corresponding color track using <a>depth map</a>'s
+          <a>depthToVideoTransform</a> field <a>videoDeviceId</a> - it
+          represents the target color video deviceID that should be used as a
+          constraint with [[!GETUSERMEDIA]] call to get the corresponding color
+          video stream track. After that, we use color track
+          <a>getSettings()</a> to access <a href=
+          "#mediatracksettings-dictionary">MediaTrackSettings</a>.
+        </p>
+        <p>
+          Let `fx_c` and `fy_c` be color track's horizontal and vertical
```

Comment:
`<a>color stream track</a>'s <a>horizontal focal length</a> and <a>vertical focal length</a> respectively.`